src/components/Auth/SignInForm/SignInForm.jsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import * as Styled from 'components/SignupForm.style';
+import AuthApiService from 'api/auth';
+import { loginValidator } from 'utils/validator';
+import { ROUTES } from 'constants/route';
+import { LOCALSTORAGE } from 'constants/localstorage';
+
+const SignInForm = () => {
+  const navigate = useNavigate();
+  const [inputs, setInputs] = useState({
+    email: '',
+    password: '',
+  });
+
+  // 로그인 input validations
+  const [validations, setValidations] = useState({
+    isValidEmail: false,
+    isValidPassword: false,
+  });
+
+  // input 유효성 검사 만족하지 않을 경우, 로그인 버튼 비활성화
+  const isDisabled = !validations.isValidEmail || !validations.isValidPassword;
+
+  // 로그인 input 유효성 검사
+  const handleChangeInputs = e => {
+    const { value, name } = e.target;
+    setInputs(prev => ({
+      ...prev,
+      [name]: value,
+    }));
+    setValidations(loginValidator({ ...inputs, [name]: value }));
+  };
+
+  const handleSignIn = async e => {
+    e.preventDefault();
+    const { email, password } = inputs;
+    try {
+      const signInResponse = await AuthApiService.signIn({ email, password });
+      // 토큰 등록
+      localStorage.setItem(LOCALSTORAGE.ACCESS_TOKEN, signInResponse.access_token);
+      // 리다이렉트
+      alert('로그인 되었습니다.');
+      navigate(0);
+    } catch (error) {
+      alert(error.response.data.message);
+    }
+  };
+
+  return (
+    <Styled.Container>
+      <Styled.Title>회원가입</Styled.Title>
+      <Styled.Form onSubmit={e => handleSignIn(e)}>
```

Comment:
handleSubmit이 event 객체만 전달받는다면 다음과 같이 바꿔볼 수 있지 않을까요?
```suggestion
      <Styled.Form onSubmit={handleSignIn}>
```

Reply:
그렇네요 수정해 보겠습니다!